src/csv.ml

```diff
@@ -489,15 +489,19 @@ let fold_right ~f ic a0 =
   List.fold_left (fun a r -> f r a) a0 lr
 
 
-let load ?separator ?excel_tricks fname =
+let load ?separator ?excel_tricks ?(headers=true) fname =
   let fh = if fname = "-" then stdin else open_in fname in
   let csv = of_channel ?separator ?excel_tricks fh in
   let t = input_all csv in
   close_in csv;
-  t
+  match headers with
+  | true -> t
+  | false -> List.tl t
 
-let load_in ?separator ?excel_tricks ch =
-  input_all (of_channel ?separator ?excel_tricks ch)
+let load_in ?separator ?excel_tricks ?(headers=true) ch =
+  match headers with
+  | true -> input_all (of_channel ?separator ?excel_tricks ch)
+  | false -> List.tl (input_all (of_channel ?separator ?excel_tricks ch))
```

Comment:
You could factor out the common subexpression so that the difference between the two branches is obvious:
```ocaml
let csv = input_all (of_channel ?separator ?excel_tricks ch) in
if headers then csv else List.tl csv
```
Also, do we really want to raise an exception if there are no headers?  if the file is empty and `headers=false`, it may be more sensible to return `[]`.

BTW, I misunderstood the meaning of `headers` in my previous comment — leaving the default to `true` is OK.  Maybe an option `rm_headers` would be more intuitive?